matlab-src/TorqueBalancing_js/centroidalTransformationT_TDot.m

```diff
@@ -27,12 +27,11 @@
 dMb     = [zeros(3),Sf(mdr)';
            Sf(mdr),zeros(3)];
 
-invMb   =  eye(6)/Mb;
-inv_dMb = -invMb*dMb*invMb;
+inv_dMb = -Mb\dMb/Mb;
```

Comment:
Probably I am just a matlab newbie, but a little comment on what is happening here it would be helpful, something like:
~~~
% dMb^{-1} = - Mb^{-1} dMb Mb^{-1}
inv_dMb = -Mb\dMb/Mb;
~~~